the-collab-lab/tcl-44-smart-shopping-list
# Update favicon and add meta data

## Description

- Added lemon slice in brand colour as a favicon.
- Added image and meta data.

## Related Issue

Closes #60 60

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

![favicon](https://user-images.githubusercontent.com/29425781/170857476-cf184075-81b3-46ed-8bd8-f03c12251972.png)

## Testing Steps / QA Criteria

- From your terminal, pull down this branch with `git pull origin dd-add-meta-data` and check that branch out with `git checkout dd-add-meta-data`
- run npm start
